src/NCrypt/NCrypt.cs

```diff
@@ -119,7 +119,7 @@ public static partial class NCrypt
         public static extern unsafe SECURITY_STATUS NCryptKeyDerivation(
             SafeKeyHandle hKey,
             NCryptBufferDesc* pParameterList,
-            out byte* pbDerivedKey,
+            byte* pbDerivedKey,
```

Comment:
This is an error in the original method that was caught in the process of overload code generation because the generated code had compile errors in it.